pkg/kubelet/kubelet.go

```diff
@@ -504,7 +504,8 @@ func NewMainKubelet(
 		hostname,
 		klet.podManager,
 		klet.kubeClient,
-		klet.volumePluginMgr)
+		klet.volumePluginMgr,
+		klet.containerRuntime)
```

Comment:
FWIW, I still think it's better to pass the kl.podCache here because otherwise, the code would be calling `docker ps` 10 times per second while waiting for kubelet to stop the container.